Update IR filter removal (#3939)
raspberrypi/documentation: documentation/asciidoc/accessories/camera/filters.adoc

image::images/hoyacm500.png[CM500 Transmission Graph,width="65%"]

== IR Filter

> Both the High Quality Camera and Global Shutter Camera contain an IR filter, which is used to reduce the camera's sensitivity to infrared light. This ensures that outdoor photos look more natural. However, some nature photography can be enhanced with the removal of this filter; the colours of sky, plants, and water can be affected by its removal. The camera can also be used without the filter for night vision in a location that is illuminated with infrared light.

Reviewer: Suggestion: clean up the introduction of the second sentence. trim down the length of the third sentence. convert the rest to a scannable unordered list.

```suggestion
Both the High Quality Camera and Global Shutter Camera contain an IR filter, which is used to reduce the camera's sensitivity to infrared light. This helps outdoor photos look more natural. You can remove the filter to:

* enhance colours in certain types of photography, such as images of plants, water, and the sky
* provide night vision in a location that is illuminated with infrared light

```

=== Filter Removal

> The filter removal procedure applies to both the HQ and GS cameras. The HQ camera is shown in the demonstration below.

Reviewer: Suggestion: the 'filter removal procedure' sounds like something you ought to ask a doctor about. maybe fewer word do trick?

```suggestion
You can remove the filter from both the HQ and GS cameras. The HQ camera is shown in the demonstration below.
```

WARNING: Before proceeding read through all of the steps and decide whether you are willing to void your warranty. *Do not proceed* unless you are sure that you are willing to void your warranty.

> WARNING: *This procedure cannot be reversed:* the adhesive that attaches the filter will not survive being lifted and replaced, and while the IR filter is about 1.1mm thick, it may crack when it is removed. *Removing it will void the warranty on the product*. Nevertheless, removing the filter will be desirable to some users.

Reviewer: I strongly suggest we don't put two admonition blocks up against each other, especially the same type. We should probably use less in general.

Personally, I would delete the first warning (on line 41). The second warning covers that info in a shorter, clearer way anyway.

image:images/FILTER_ON_small.jpg[width="65%"]

> Work in a clean and dust-free environment, as the sensor will be exposed to the air. Unscrew the two 1.5 mm hex lock keys on the underside of the main circuit board. Be careful not to let the washers roll away.

Reviewer: Suggestion: perhaps we could convert this to an ordered list, since these are steps that must happen in a sequence? Then we could lift the 'clean and dust-free environment' recommendation into the list introduction.

== IR Filter

Both the High Quality Camera and Global Shutter Camera contain an IR filter to reduce the camera's sensitivity to infrared light and help outdoor photos look more natural. However, you may remove the filter to:

* Enhance colours in certain types of photography, such as images of plants, water, and the sky
* Provide night vision in a location that is illuminated with infrared light

=== Filter Removal

WARNING: *This procedure cannot be reversed:* the adhesive that attaches the filter will not survive being lifted and replaced, and while the IR filter is about 1.1mm thick, it may crack when it is removed. *Removing it will void the warranty on the product*. Nevertheless, removing the filter will be desirable to some users.

You can remove the filter from both the HQ and GS cameras. The HQ camera is shown in the demonstration below.

image:images/FILTER_ON_small.jpg[width="65%"]

NOTE: Make sure to work in a clean and dust-free environment, as the sensor will be exposed to the air. 

> 1. Unscrew the two 1.5 mm hex lock keys on the underside of the main circuit board. Be careful not to let the washers roll away.

Reviewer: Instead of writing step numbers manually, prefer just a dot (`.`). Asciidoctor still recognises this as an ordered list, and will automatically generate the right numbers for you.

This way, you won't end up with cascading multi-line changes in a set of steps if you remove or add a step.

Reviewer: If this isn't in the style guide already, it should be :-)

+
image:images/SCREW_REMOVED_small.jpg[width="65%"]
2. There is a gasket of slightly sticky material between the housing and PCB which will require some force to separate. You may try some ways to weaken the adhesive, such as a little isopropyl alcohol and/or heat (~20-30 C). 
3. Once the adhesive is loose, lift up the board and place it down on a very clean surface. Make sure the sensor does not touch the surface.
+
image:images/FLATLAY_small.jpg[width="65%"]

> 4. Turn the lens mount around so that it is "looking" upwards and place it on a table.

Reviewer: Suggestion: scare quotes are against the style guide (not your fault here, of course). Maybe say "Face the lens upwards and place the mount on a flat surface" instead?
